JsonSchema/Formats.cs

```diff
@@ -272,8 +273,7 @@ private static bool CheckDateFormat(JsonNode? node, params string[] formats)
 		//We use a fallback to catch these cases
 
 		//from built from https://regex101.com/r/qH0sU7/1, edited to support all date+time examples in https://ijmacd.github.io/rfc3339-iso8601/
-		var regex = new Regex(@"^((?:(\d{4}-\d{2}-\d{2})([Tt_]| )(\d{2}:\d{2}:\d{2}(?:\.\d+)?))([Zz]|[\+-]\d{2}:\d{2}))$");
-		var match = regex.Match(dateString);
+		var match = _dateTimeRegex.Match(dateString);
```

Comment:
This is likely the reason there was a greater improvement for my scenario compared to `JsonSchema.Benchmark`, as I'm validating a date time in each call to `Evaluate`.

Reply:
Please move the comment on 275 up with the regex definition.